index.html

```diff
@@ -6,7 +6,411 @@
+          </li>
+          <li class="categories-item">
+            <a class="categories-link" href="#"><img class src="" alt="Умные часы"></a><br>
+            <h3 class="categories-name">Умные часы</h3>
+          </li>
+          <li class="categories-item">
+            <a class="categories-link" href="#"><img class src="" alt="Квадракоптер с видеокамерой"></a><br>
+            <h3 class="categories-name">Квадракоптеры</h3>
+          </li>
+        </ul>
+      </section>
+
+      <section class="services">
+        <h2 class="visually-hidden">Услуги</h2>
+        <ul class="services-list">
+          <li class="services-item">
+            <button class="services-button" type="button">
+              Доставка
+            </button><br>
+            <h3 class="services-name">Доставка</h3>
+            <p class="services-description">
+              Мы с удовольствием доставим ваш товар прямо<br>
+              к вашему подъезду совершенно бесплатно! Ведь<br>
+              мы неплохо заработаем, поднимая его на ваш этаж.
+            </p>
+          </li>
+          <li class="services-item">
+            <button class="services-button" type="button">
+              Гарантия
+            </button>
+            <h3 class="services-name">Гарантия</h3>
+            <p class="services-description">
+              Если из-за возгорания купленного у нас товара у вас<br>
+              сгорит дом — не переживайте, мы выдадим вам новый.<br>
+              Товар, не дом, конечно же.
+            </p>
+          </li>
+          <li class="services-item">
+            <button class="services-button" type="button">
+              Кредит
+            </button>
+            <h3 class="services-name">Кредит</h3>
+            <p class="services-description">
+              Залезть в долговую яму стало проще! Кредитные<br>
+              консультанты подберут для вас наиболее выгодные<br>
+              условия кредита. Выгодные для банка, разумеется.
+            </p>
+          </li>
+        </ul>
+      </section>
+
+      <aside class="rare_order">
+        <h2 class="rare_order-title">
+          Доставим редкий заказ под заказ
+        </h2>
+        <a class="rare_order-info" href="#">
+          <span class="visually-hidden">
+            Информация
+          </span>
+        </a>
+      </aside>
+
+      <section class="about">
+        <h2 class="about-title">О нас</h2>
+        <p class="about-description">
+          Огромный выбор гаджетов не оставит равнодушным гика, который есть в каждом из нас.
+        </p>
+        <p class="about-description">
+          Мы можем доставить ваш товар в самые отдалённые уголки России транспортными компаниями:
+        </p>
+        <ul class="about-list">
+          <li class="about-item">
+            <b class="company">
+              Меловые линии
+            </b>
+          </li>
+          <li class="about-item">
+            <b class="company">
+              СДИК
+            </b>
```

Comment:
Необязательно, просто ставлю в курс о не самом известном тэге
```suggestion
            <abbr title="Служба доставки Икспресс-Курьер">
              СДИК
            </abbr>
```